SVN commit 1240130 by teran:

rolling back change made by r1235446, it broke keybindings entirely.

BUG: 277020

http://websvn.kde.org/?view=revision&revision=1240130
--- kcalc/kcalc.ui	2011/06/05 18:25:18	1235446
+++ kcalc/kcalc.ui	2011/07/06 18:27:58	1240130
@@ -616,7 +616,7 @@
              <string>7</string>
             </property>
             <property name="shortcut">
-             <string notr="true">7</string>
+             <string>7</string>
             </property>
            </widget>
           </item>
@@ -632,7 +632,7 @@
              <string>8</string>
             </property>
             <property name="shortcut">
-             <string notr="true">8</string>
+             <string>8</string>
             </property>
            </widget>
           </item>
@@ -648,7 +648,7 @@
              <string>9</string>
             </property>
             <property name="shortcut">
-             <string notr="true">9</string>
+             <string>9</string>
             </property>
            </widget>
           </item>
@@ -680,7 +680,7 @@
              <string>4</string>
             </property>
             <property name="shortcut">
-             <string notr="true">4</string>
+             <string>4</string>
             </property>
            </widget>
           </item>
@@ -696,7 +696,7 @@
              <string>5</string>
             </property>
             <property name="shortcut">
-             <string notr="true">5</string>
+             <string>5</string>
             </property>
            </widget>
           </item>
@@ -712,7 +712,7 @@
              <string>6</string>
             </property>
             <property name="shortcut">
-             <string notr="true">6</string>
+             <string>6</string>
             </property>
            </widget>
           </item>
@@ -728,7 +728,7 @@
              <string>1</string>
             </property>
             <property name="shortcut">
-             <string notr="true">1</string>
+             <string>1</string>
             </property>
            </widget>
           </item>
@@ -744,7 +744,7 @@
              <string>2</string>
             </property>
             <property name="shortcut">
-             <string notr="true">2</string>
+             <string>2</string>
             </property>
            </widget>
           </item>
@@ -760,7 +760,7 @@
              <string>3</string>
             </property>
             <property name="shortcut">
-             <string notr="true">3</string>
+             <string>3</string>
             </property>
            </widget>
           </item>
@@ -786,7 +786,7 @@
              <string>0</string>
             </property>
             <property name="shortcut">
-             <string notr="true">0</string>
+             <string>0</string>
             </property>
            </widget>
           </item>
